.github/CODEOWNERS

```diff
@@ -2,147 +2,133 @@
 # Each line is a case-sensitive file pattern followed by one or more owners.
 # Order is important. The last matching pattern has the most precedence.
 # More information: https://docs.github.com/en/articles/about-code-owners
+#
+# Please mirror the repository's file hierarchy in case-insensitive
+# lexicographic order.
```

Comment:
> Since the file patterns are case-sensitive, I suggest moving SwiftCompilerSources

I am not sure I follow.

Regarding the suggested change, what I meant by case-insensitive order is that uppercase letters are ordered the same as lowercase ones. So "Sema" goes before "SIL" despite the capital "I". Maybe I worded this poorly?

Reply:
Ah, OK. I find it slightly counterintuitive, but maybe I’m just used to how Finder sorts stuff.